variables.tf

```diff
@@ -86,11 +86,11 @@ variable "layers" {
 variable "datadog_python_layer_version" {
   type        = number
   description = "The version of the Datadog Python Layer"
-  default     = 53
+  default     = 54
 }
 
 variable "datadog_extension_layer_version" {
   type        = number
   description = "The version of the Datadog Extension Layer"
-  default     = 22
+  default     = 23
```

Comment:
Looks like the most recent version of this is `31` https://github.com/DataDog/datadog-lambda-extension/releases/tag/v31

```There is a conflict with V30 during publishing the new extension, so V31 is the new version after V29.

arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:31
arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension-ARM:31
arn:aws-us-gov:lambda:us-gov-<AWS_REGION>:002406178527:layer:Datadog-Extension:31
arn:aws-us-gov:lambda:us-gov-<AWS_REGION>:002406178527:layer:Datadog-Extension-ARM:31```